contracts/contracts/vault/VaultCore.sol

```diff
@@ -27,9 +27,9 @@ contract VaultCore is VaultStorage {
     using StableMath for uint256;
     using SafeMath for uint256;
     // max signed int
-    uint256 constant MAX_INT = 2**255 - 1;
+    uint256 private constant MAX_INT = 2**255 - 1;
```

Comment:
Why are these two private and the others internal?

Reply:
My thinking is no Contract is going to be extending the VaultCore.

Reply:
I'd rather just stick with internal, it shouldn't matter gas or contract size wise.

Reply:
ok sounds good